includes/Editor.php

```diff
@@ -16,6 +16,8 @@
  */
 class Editor {
 
+	const ALLOWED_POST_TYPES = array( 'post', 'page', 'neve_custom_layouts' );
```

Comment:
can you make the list filterable using something like tpc_cloud_post_types

Reply:
I extended the `ALLOWED_POST_TYPES` to be filtered, however since the save is handled inside JS via API here: https://github.com/Codeinwp/templates-patterns-collection/pull/222/files#diff-8f3e62672e7b7cfcb47331b1fc46d42f60e832f8550fa4762df2f33694a3e61aL393-R417

An initialization for individual custom post types will need to be added if we want to extend this in the future. This makes the filter a bit useless at the moment.